iOS/PixelDefinitions/pixels/widget_reporting.json5

```diff
@@ -0,0 +1,42 @@
+// See https://app.asana.com/1/137249556945/project/392891325557410/task/1210882421460693
+{
+    'm_widget-report': {
```

Comment:
doesn't it miss _ios suffix? (applies to both pixels added)

Reply:
Good catch